cmd/rook/nfs/operator.go

```diff
@@ -33,8 +33,12 @@ https://github.com/rook/rook`,
 }
 
 func init() {
-	flags.SetFlagsFromEnv(operatorCmd.Flags(), rook.RookEnvVarPrefix)
-	flags.SetLoggingFlags(operatorCmd.Flags())
+	if err := flags.SetFlagsFromEnv(operatorCmd.Flags(), rook.RookEnvVarPrefix); err != nil {
+		panic(err)
+	}
+	if err := flags.SetLoggingFlags(operatorCmd.Flags()); err != nil {
```

Comment:
The same applies here to panic from inside SetLoggingFlags()

Reply:
Done.